templates/flavors/machinepool/machine-pool-deployment.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: exp.cluster.x-k8s.io/v1alpha3
+kind: MachinePool
+metadata:
+  name: "${CLUSTER_NAME}-mp-0"
+spec:
+  clusterName: "${CLUSTER_NAME}"
+  replicas: ${WORKER_MACHINE_COUNT}
+  template:
+    spec:
+      clusterName: "${CLUSTER_NAME}"
+      version: "${KUBERNETES_VERSION}"
+      bootstrap:
+        configRef:
+          name: "${CLUSTER_NAME}-mp-0"
+          apiVersion: bootstrap.cluster.x-k8s.io/v1alpha3
+          kind: KubeadmConfig
+      infrastructureRef:
+        name: "${CLUSTER_NAME}-amp-0"
+        apiVersion: exp.infrastructure.cluster.x-k8s.io/v1alpha3
+        kind: AzureMachinePool
+---
+apiVersion: exp.infrastructure.cluster.x-k8s.io/v1alpha3
+kind: AzureMachinePool
+metadata:
+  name: "${CLUSTER_NAME}-amp-0"
```

Comment:
I noticed that for regular VMs, we name the VMs and nodes the same as the MachineDeployment, `${CLUSTER_NAME}-md-0`, should we do the same for machine pools and call this `${CLUSTER_NAME}-mp-0` instead?

Reply:
Sure.